cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils.go

```diff
@@ -63,10 +63,15 @@ var (
 	errInvalidMaxAnnotation = errors.New("invalid max annotation")
 
 	// machineDeleteAnnotationKey is the annotation used by cluster-api to indicate
-	// that a machine should be deleted. Because this key can be affected by the
+	// that a machine should be preferred for deletion. Because this key can be affected by the
 	// CAPI_GROUP env variable, it is initialized here.
 	machineDeleteAnnotationKey = getMachineDeleteAnnotationKey()
 
+	// markMachineForDelete is the annotation used by cluster-api to indicate
+	// that a machine should be deleted ASAP. Because this key can be affected by the
+	// CAPI_GROUP env variable, it is initialized here.
+	markMachineForDeleteAnnotationKey = getMarkMachineForDeleteAnnotationKey()
```

Comment:
I'm adding this at the recommendation of the cluster-api maintainers to avoid breaking the existing behavior of the `delete-machine` annotation.